HabboHotel/Users/HabboStatsService.cs

```diff
@@ -0,0 +1,69 @@
+using Dapper;
+using Plus.Database;
+
+namespace Plus.HabboHotel.Users;
+
+public class HabboStatsService : IHabboStatsService
+{
+    private readonly IDatabase _database;
+
+    public HabboStatsService(IDatabase database) => _database = database ?? throw new ArgumentNullException(nameof(database));

```

Comment:
`?? throw new ArgumentNullException(nameof(database))` is not needed as the emulator wouldnt boot if the database is not available.